mito-ai/style/ChatTaskpane.css

```diff
@@ -103,3 +107,87 @@
 .message-edit-buttons button:hover {
     background: #f0f0f0;
 }
+
+.chat-dropdown {
+    position: absolute;
+    width: 100%;
+    z-index: 9999;
+}
+
+.chat-dropdown.above {
+    bottom: 100%;
+    margin-bottom: 5px;
+}
+
+.chat-dropdown.below {
+    top: 100%;
+    margin-top: 5px;
+}
+
+.chat-dropdown {
+    display: flex;
+    justify-content: space-evenly;
+    align-items: baseline;
+}
+
+.chat-dropdown-list {
+    position: relative;
+    border: 0px;
+    border-radius: 5px;
+    background-color: white;
+    list-style-type: none;
+    padding: 0;
+    margin: 0;
+    width: 100%;
+    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.2);
+}
+
+.chat-dropdown-item {
+    padding: 8px;
+    cursor: pointer;
+    display: flex;
+    align-items: center;
+    gap: 8px;
+}
+
+.chat-dropdown-item:hover {
+    background-color: var(--item-selected-color);
+}
+
+.chat-dropdown-item.selected {
+    background-color: var(--item-selected-color);
+}
+
+.chat-dropdown-item:first-child {
+    /* Add rounded corners to the top of the first selected dropdown item */
+    border-top-left-radius: 5px;
+    border-top-right-radius: 5px;
+}
+
+.chat-dropdown-item:last-child {
+    /* Add rounded corners to the bottom of the last selected dropdown item */
+    border-bottom-left-radius: 5px;
+    border-bottom-right-radius: 5px;
+}
+
+.chat-dropdown-item-type {
+    color: var(--muted-text-color);
+    font-family: var(--jp-code-font-family);
+    min-width: 35px; /* Fixed width for type column */
+}
+
+.chat-dropdown-item-name {
```

Comment:
This is getting too cut off when both the column name is long and the dataframe name is long. 
<img width="1312" alt="Screenshot 2024-11-18 at 11 18 55 PM" src="https://github.com/user-attachments/assets/b67451ef-7832-443e-a792-6145e82eb69e">